apps/admin/frontend/src/components/export_election_ballot_package_modal_button.test.tsx

```diff
@@ -20,6 +21,8 @@ import { ApiMock, createApiMock } from '../../test/helpers/api_mock';
 let apiMock: ApiMock;
 
 beforeEach(() => {
+  MockDate.set(new Date(2023, 0, 1));
+
```

Comment:
I believe this conflicts with #3496 where @eventualbuddha removes our use of `mockdate` in favor of a `jest` utility.

Reply:
Okay, took a closer look at this and https://github.com/votingworks/vxsuite/pull/3496, and using `jest`'s `setSystemTime` utility would require me to upgrade the version of Jest (like https://github.com/votingworks/vxsuite/pull/3496 does), which balloons scope a bit. And we've since closed https://github.com/votingworks/vxsuite/pull/3496 anyway since it didn't fix test flakiness. So I'm going to leave this as is